equilibrium.yaml

```diff
@@ -0,0 +1,35 @@
+specVersion: 1.0.0
+name: subquery-nova-equilibrium
+version: 1.0.0
+runner:
+  node:
+    name: '@subql/node'
+    version: '*'
+  query:
+    name: '@subql/query'
+    version: v1.5.0
+description: Nova SubQuery project
+repository: https://github.com/nova-wallet/subquery-nova
+schema:
+  file: ./schema.graphql
+network:
+  chainId: '0x89d3ec46d2fb43ef5a9713833373d5ea666b092fa8fd68fbc34596036571b907'
+  endpoint: >-
+    wss://equilibrium-rpc.dwellir.com
```

Comment:
Probably Dwellir is not the best choice. I see also a node by Equilibrium in polkadot js. Probably better to go with this one.

Reply:
Unfortunately, it's not a case for us, as only dwellir provide archive node
<img width="300" alt="Screenshot 2023-04-26 at 11 26 13" src="https://user-images.githubusercontent.com/40560660/234516346-746e20ee-a130-424a-9175-caec39212d19.png">